packages/main/src/Breadcrumbs.js

```diff
@@ -516,12 +538,43 @@ class Breadcrumbs extends UI5Element {
 	 */
 	get _linksData() {
 		const items = this.getSlottedNodes("items").slice(this._overflowSize);
+		const itemsCount = items.length; // get size before removing of current location
 
 		if (this._endsWithCurrentLocationLabel) {
 			items.pop();
 		}
 
-		return items.filter(item => this._isItemVisible(item));
+		return items
+			.filter(item => this._isItemVisible(item))
+			.map((item, index) => {
+				const { accessibleNameRef, accessibleName } = this._getItemAccessibleNameData(item, index + 1, itemsCount);
+
+				item.accessibleNameRef = accessibleNameRef;
```

Comment:
lets be consistent and name this property _accessibleNameRef as other

Reply:
AccessibleNameRef usage is removed now.